Whitelist-Dapp/hardhat-tutorial/contracts/Greeter.sol

```diff
@@ -0,0 +1,22 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
Relicat ?

Reply:
What is the purpose of this SC ?

Reply:
Hey this contract comes from the hardhat package, we can delete it :)